src/bp/ui-studio/src/web/views/Content/index.tsx

```diff
@@ -41,7 +52,11 @@ class ContentView extends Component<Props, State> {
 
     if (this.canRead) {
       this.props.fetchContentCategories()
+      this.props.fetchFlows()
       this.fetchCategoryItems(this.state.selectedId)
+      getQnAContentElementUsage().then(res => {
+        this.setState({ qnaUsage: res.data })
+      })
```

Comment:
The usage is a bit weird, can you implement that call using Redux ? You can take inspiration from intentReceived in the actions file

Reply:
Using Redux is really difficult (at least for now), I hope it is done right.